src/javascript/app/base/page.js

```diff
@@ -68,7 +68,7 @@ const Page = (() => {
                     // no default
                 }
             });
-            scrollToTop();
+            // scrollToTop();
```

Comment:
IMO there's no need to comment the JS files. just commenting the HTML element is enough.